action/src/build-comparadise-url.ts

```diff
@@ -1,11 +1,10 @@
 import { getInput } from '@actions/core';
 import { context } from '@actions/github';
 
-export const buildComparadiseUrl = () => {
+export const buildComparadiseUrl = (hash: string) => {
   const bucketName = getInput('bucket-name', { required: true });
-  const commitHash = getInput('commit-hash', { required: true });
   const comparadiseHost = getInput('comparadise-host');
   const { owner, repo } = context.repo;
 
-  return `${comparadiseHost}/?hash=${commitHash}&owner=${owner}&repo=${repo}&bucket=${bucketName}`;
+  return `${comparadiseHost}/?hash=${hash}&owner=${owner}&repo=${repo}&bucket=${bucketName}`;
```

Comment:
We discussed using a different parameter in the URL, but since hash is quite generic, I decided to keep it as is. Lemme know if you feel I should update it.